bigtable-hbase-parent/bigtable-hbase-integration-tests/src/test/java/com/google/cloud/bigtable/hbase/TestCheckAndMutate.java

```diff
@@ -285,4 +288,52 @@ public void testCheckAndMutate() throws IOException {
         row.containsColumn(SharedTestEnvRule.COLUMN_FAMILY, qualPut));
     table.close();
   }
+
+  @Test
+  public void testCompareOps() throws IOException {
```

Comment:
I think we should add tests for CompareOp.EQUAL and CompareOp.NOT_EQUAL as well.